azurerm/resource_arm_cosmosdb_mongo_database.go

```diff
@@ -131,10 +173,22 @@ func resourceArmCosmosDbMongoDatabaseRead(d *schema.ResourceData, meta interface
 		return fmt.Errorf("Error reading Cosmos Mongo Database %s (Account %s): %+v", id.Database, id.Account, err)
 	}
 
-	d.Set("resource_group_name", id.ResourceGroup)
-	d.Set("account_name", id.Account)
+	_ = d.Set("resource_group_name", id.ResourceGroup)
+	_ = d.Set("account_name", id.Account)
```

Comment:
We don't need the assingment
```suggestion
	d.Set("account_name", id.Account)
```